events/schema.py

```diff
@@ -598,13 +602,55 @@ class Meta:
         node = InternalOrTicketSystemEnrolmentUnion
 
 
+class OccurrenceArrivalStatusNode(ObjectType):
+    enrolment_count = graphene.Int(
+        required=True,
+        description="**DEPRECATED:** Total number of enrolments for this occurrence.",
+        deprecation_reason="This field exposes potentially sensitive data "
+        "and will be removed in a future release. "
+        "Consider using a more secure method to access this information.",
+    )
+    attended_enrolment_count = graphene.Int(
+        required=True,
+        description="**DEPRECATED:**  Number of enrolments marked as attended "
+        "for this occurrence.",
+        deprecation_reason="This field exposes potentially sensitive data "
+        "and will be removed in a future release. "
+        "Consider using a more secure method to access this information.",
+    )
```

Comment:
I don't think the enrolment counts are sensitive information, so they can be seen by everyone IMHO. So keeping them open is simply easier. Of course this is a judgment call, but really, what is the worst that could happen if these are kept openly accessible?

Reply:
At least they should be available without reading a ticket. Now only way to get the status anonymously is to read a ticket.

Reply:
I do agree that "sensitive" is not the best word, but was the best I could come up with.